dist/css/bootstrap.min.css

```diff
@@ -3,5 +3,5 @@
  * Copyright 2011-2017 The Bootstrap Authors
```

Comment:
You shouldn't include dist files in a PR.

Reply:
Yes, I noticed that after I did it.
But, now, is there a way to remove the dist files of my commit or I need to create a new branch with my changes (without dist) and a new PR?

Thanks!

Reply:
I'm afraid that a new commit with deletion of the /dist can delete the /dist of v4-dev branch after merge

Reply:
`git rebase -i` read about it.

Reply:
@XhmikosR Thanks, I got it!